examples/update-demo/kitten-rc.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ReplicationController
+id: update-demo-kitten
```

Comment:
I want a separate schema for the replacement controller, so it can omit #replicas and inherit #replicas from the running controller. Doesn't seem to be a benefit to further parameterizing unless we are merging the 2 schemas.